instrumentation/basic_logger.go

```diff
@@ -86,6 +86,29 @@ func Stringable(key string, value fmt.Stringer) *Field {
 	return &Field{Key: key, String: value.String(), Type: StringType}
 }
 
+func StringableSlice(key string, values interface{}) *Field {
+	builder := strings.Builder{}
```

Comment:
Here lies a problem: I think we need to store the whole string slice and not form a string, because we are going to format the slice differently in both human readable formatter and JSON formatter.

Please consider introducing a `[]string` array to the `Field` type and storing it there and implementing support on `Formatter` level as well.

Reply:
got it, so for humans we can do sort of a json-lint like format and for the machine logger just compressed json - will change